launch/schunk_neck.launch

```diff
@@ -0,0 +1,18 @@
+<launch>
+  <!-- Load the controllers -->
+  <rosparam command="load"
+            file="$(find herb_launch)/config/schunk_controllers.yaml"/>
+
+  <node name="schunk_pw70_ros" pkg="schunk_pw70_ros" type="schunk_robot" output="screen"/>
+
+  <!-- TODO timeout? -->
+  <node name="ros_control_manager"
```

Comment:
Launching both the WAM and neck control managers didn't work out of the box. I think that's because they share the same name, but didn't have time to test.